warehouse/ingest-core/src/main/java/datawave/ingest/util/NGramTokenizationStrategy.java

```diff
@@ -130,48 +126,30 @@ public int tokenize(final NormalizedContentInterface content, int maxNGramLength
                     }
                     
                     // Create the tokenizer and tokenizing stream
-                    tokenizer = new NGramTokenizer(2, maxNGramLength);
-                    tokenizer.setReader(reader);
-                    tokenizer.reset();
-                    tokenStream = new ClassicFilter(tokenizer);
-                    tokenStream = new LowerCaseFilter(tokenStream);
-                    tokenStream.addAttribute(CharTermAttribute.class);
-                    
-                    // Reset the n-gram count
-                    ngramCount = 0;
-                    
-                    // Increment the tokenizer and applied any generated n-grams
-                    String ngram = null;
-                    while (null != (ngram = this.increment(tokenizer))) {
-                        if (this.updateFilter(ngram, content)) {
-                            ngramCount++;
+                    try (NGramTokenizer tokenizer = new NGramTokenizer(2, maxNGramLength)) {
+                        tokenizer.setReader(reader);
+                        tokenizer.reset();
+                        try (TokenStream tokenStream = new ClassicFilter(tokenizer)) {
+                            try (TokenStream lcfTokenStream = new LowerCaseFilter(tokenStream)) {
+                                lcfTokenStream.addAttribute(CharTermAttribute.class);
+                                
+                                // Reset the n-gram count
+                                ngramCount = 0;
+                                
+                                // Increment the tokenizer and applied any generated n-grams
+                                String ngram = null;
+                                while (null != (ngram = this.increment(tokenizer))) {
+                                    if (this.updateFilter(ngram, content)) {
+                                        ngramCount++;
+                                    }
+                                }
+                            }
                         }
                     }
                 } catch (final TokenizationException e) {
-                    exception = e;
                     throw e;
                 } catch (final IOException e) {
-                    exception = new TokenizationException(e);
-                    throw exception;
-                } finally {
-                    // Close the tokenizer
-                    if (null != tokenizer) {
-                        try {
-                            tokenizer.end();
```

Comment:
It appears that tokenizer.end() is no longer called with this change.  We might want to consider how best to handle that.  Could we add a call to the end method to the close method?

Reply:
So with the 'try-with-resources' it will automatically call close().  Since this object is just being made here -- what does the end() call do?
I can just revert the whole block if it is really needed.